packages/flutter/test/widgets/shortcuts_test.dart

```diff
@@ -1419,4 +1307,526 @@ void main() {
       await tester.sendKeyUpEvent(LogicalKeyboardKey.keyB);
     });
   });
+
+  group('ShortcutRegistrar', () {
+    testWidgets('trigger ShortcutRegistrar on key events', (WidgetTester tester) async {
+      int invokedA = 0;
+      int invokedB = 0;
+      await tester.pumpWidget(
+        ShortcutRegistrar(
+          child: TestCallbackRegistration(
+            shortcuts: <ShortcutActivator, Intent>{
+              const SingleActivator(LogicalKeyboardKey.keyA): _VoidCallbackIntent(() {
```

Comment:
Can we replace _VoidCallbackIntent with the same class from the other PR?

Reply:
It's landed. I updated it.